src/main/java/com/pk/writer/BibTexWriter.java

```diff
@@ -0,0 +1,82 @@
+package com.pk.writer;
+
+import com.pk.model.Book;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.Month;
+
+public class BibTexWriter implements BooksWriter {
+
+    @Override
+    public void write(Book[] books, String path) {
+        //name of the file should have .bib extension
+        String format = "@book";
+        try(
+                FileWriter file = new FileWriter(path);
+                BufferedWriter writer = new BufferedWriter(file)
+        ) {
+            for(Book book: books) {
+                writer.write(format + "{book,\n");
```

Comment:
1. Można by tu zrobić metodę w BookService np bookToBibtexString. Wrzucasz książkę do metody, dostajesz String i ten String zapisujesz.
2. Jak będziesz łączyć Stringi to użyj StringBuilder. Jest dużo optymalniejszy.